documentation/introduction/introduction-overview.asciidoc

```diff
@@ -3,42 +3,18 @@ title: Overview
 order: 1
 layout: page
 ---
+
 = Overview
+
 This chapter provides an overview of the Vaadin architecture and introduces the key concepts in the framework.
 
 == What is Vaadin?
 
-On the other side, the creation of Java Component UI API on the server side can be difficult for JavaScript developers, thus, Vaadin allows creating TypeScript views on the client-side. In this case, the application can be started without creating a UI instance in the server side,
-which means that applications might be stateless.
-
-TIP: Watch the https://vaadin.com/learn/training/v14-intro[Vaadin 14: Intro] free training video to learn more about the Vaadin framework, basic Vaadin application architecture and how Vaadin components work.
-
-Vaadin includes:
-
-* A type-safe Java UI Component API on the server side that facilitates the use of the Web Components.
-
-* A TypeScript view on the client side using link:https://lit-element.polymer-project.org/[LitElement^] as a helper library.
+Vaadin is a Java user interface framework coupled with JavaScript web components.
+You can create web apps in server-driven Java, or in JavaScript, by using the Vaadin web components with any backend.
+// NOTE Update in V15: TypeScript, Java backend.
 
-* Automated bi-directional communication between the server and the browser, that:
-
-** Gives Java developers full access to all modern web enhancements.
-** Makes it easier to connect the UI to data via a robust Java backend, instead of using traditional REST-based communication.
-** Lets you choose either basic HTTP(S) or WebSocket communication protocol for building real-time applications.
-** Provides JavaScript developers full access to the backend from TypeScript by using the Vaadin endpoint.
-* Two-way data binding: when the UI changes on either the client or the server, the changes automatically reflect on the other side.
-
-image:images/v10-architecture.svg[Vaadin 10 Architecture,1200,430]
-
-Vaadin allows you to access browser APIs, Web Components, and even simple DOM elements, directly from the server-side Java,
-or access the Java DTO class and generate the corresponding TypeScript module from the client side. It is not necessary to understand how the client-to-server communication or Web Components work. This leaves you free to focus on creating components that work at a higher-abstraction level.
-
-image:images/dom-to-java.svg[dom-to-java,500,432]
-
-== Building TypeScript views with Components
-
-To build an application using TypeScript view, please follow <<../typescript/intro-to-typescript-in-v15#add-typescript-view, the Adding a TypeScript View tutorial>>.
```

Comment:
Should content like this which talks about TypeScript be included somehow as well? Otherwise, TypeScript seems to be overlooked.

Reply:
Well OK, maybe it's best to put that and the other material there for now.

I'm intending to do a bigger revision to include more about TypeScript, so this was just to revert the changes. But OK, it could take some days so maybe it's best those passages there.

Reply:
I dropped those sub-sections from the page to make it simpler earlier, so there's no place for that anymore. The reader can navigate the menu to TypeScript, as can navigate to Java. (Well they are not equivalent currently, but will be.)